dev_setup.py

```diff
@@ -55,14 +60,16 @@ def write_files(api_key, env_path, total_clients):
 
     # Get user input for API key
     api_key = input("Enter your API key from regulations.gov: ")
+    validator_api_key = input("Enter a different API key from regulations.gov for the validator: ")
```

Comment:
This may not be possible for the dev team who only has one key available.

Reply:
Good point, I figured it wouldn't overload the one API key that they would use on everything else if they had a different one, but it might not always be available.